examples/2d/elixir_euler_unstructured_quad.jl

```diff
@@ -0,0 +1,60 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations2D(1.4)
+num_eqns = nvariables(equations)
+
+initial_condition = initial_condition_convergence_test
+source_terms = source_terms_convergence_test
+boundary_conditions = boundary_condition_convergence_test
+
+###############################################################################
+# Get the DG approximation space
+
+poly_deg = 5
+surface_flux = flux_hll # flux_lax_friedrichs
+solver = DGSEM(poly_deg, surface_flux)
+
+###############################################################################
+# Get the curved quad mesh from a file
+
+mesh_file = "./examples/2d/BoxAroundCircle8.mesh"
```

Comment:
This should get a relative path w.r.t the location of the elixir, e.g. using `@__DIR__`.